pkgs/tools/admin/trinsic-cli/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, rustPlatform, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "trinsic-cli";
+  version = "1.1.0";
+
+  src = fetchurl {
+    url = "https://github.com/trinsic-id/sdk/releases/download/v${version}/${pname}-1.1.2.tar.gz";
+    sha256 = "sha256:0bbhwihzw6yy26d4sf9q8v0b1c5xkizszlabnm5ldihjb7wyy8r5";
+  };
+
+  nativeBuildInputs = [
+    rustPlatform.rust.cargo
+    rustPlatform.rust.rustc
+  ];
+
+  unpackPhase = ''
+    tar xvzf $src
+    ls -la trinsic-cli
+  '';
+
+  buildPhase = ''
+    cd trinsic-cli
+    cargo vendor
+    cargo build --release
+  '';
```

Comment:
Can be replaced by changing the directory in preBuild. And letting the defaults compile the program - so no need for running cargo manually